src/stack.ts

```diff
@@ -2017,6 +2017,9 @@ export class Stack {
 
   private async bindLeftoverAssets(queries: IQuery, locale: string, pointerList: IShelf[]) {
     // const contents = await readFile(getAssetsPath(locale) + '.json')
+    if (!this.sanitizeIQuery(queries)) {
+      throw new Error('Invalid queries provided');
+    }
```

Comment:
@harshithad0703 , have you added this condition to fix test cases ?

Reply:
@aman19K this condition is added for Snyk UI : Fail issue reported by SRE

Reply:
@harshithad0703 , then it's better to convert this PR to draft and let @AniketDev7 first verify the changes. Pls explain him about fixes.
Hope you've already verified all changes. 

CC: @netrajpatel